packages/firestore/src/api/database.ts

```diff
@@ -676,7 +693,9 @@ export class Firestore implements PublicFirestore, FirebaseService {
   runTransaction<T>(
     updateFunction: (transaction: PublicTransaction) => Promise<T>
   ): Promise<T> {
-    return this.ensureClientConfigured().transaction(
+    this.ensureClientConfigured();
```

Comment:
Assign to a temporary rather than referring to `this._firestoreClient!`?

Reply:
Done